omod/src/main/java/org/openmrs/web/dwr/DWRPatientService.java

```diff
@@ -565,7 +566,32 @@ public String exitPatientFromCare(Integer patientId, Integer exitReasonId, Strin
 						ret = "Unable to locate cause of death in dictionary - cannot proceed";
 					}
 				}
+				
```

Comment:
Same here - if this is code that we are essentially just adding back in, can you indicate from where?